packages/framework/fluid-framework/README.md

```diff
@@ -1,8 +1,30 @@
 # fluid-framework
 
-The `fluid-framework` package bundles a collection of Fluid Framework client packages for easy use when paired with a corresponding service client package (ex. `@fluidframework/azure-client` & `@fluidframework/tinylicious-client`).
+The `fluid-framework` package bundles a collection of Fluid Framework client libraries for easy use when paired with a corresponding service client package (e.g. `@fluidframework/azure-client`, `@fluidframework/tinylicious-client`, or `@fluid-experimental/osdp-client (BETA)`).
 
-<!-- AUTO-GENERATED-CONTENT:START (README_DEPENDENCY_GUIDELINES_SECTION:includeHeading=TRUE) -->
+## Contents
+
+The `fluid-framework` package consists primarily of two portions: the `IFluidContainer` and a selection of distributed data structures (DDSes).
+
+### IFluidContainer
+
+The **[IFluidContainer][]** interface is the one of the types returned by calls to `createContainer()` and `getContainer()` on the service clients such as `AzureClient`.
+It includes functionality to retrieve the Fluid data contained within, as well as to inspect the state of the collaboration session connection.
+
+### DDS packages
+
+You'll use one or more DDS data structures in your container to model your collaborative data.
+The `fluid-framework` package offers the following data structures:
+
+1. **[SharedTree][]**
+1. **[SharedMap][]**, a map-like data structure for storing key/value pair data
+    - Note: as of version 2.0, `SharedMap` is deprecated. Please use `SharedTree` instead.
+
+## Tutorial
+
+Check out the Hello World tutorial using the `fluid-framework` package [here](https://fluidframework.com/docs/start/tutorial/).
```

Comment:
Nit: try to make the text of links describe what it links to instead of "here."

```suggestion
Check out the [Hello World tutorial](https://fluidframework.com/docs/start/tutorial/) using the `fluid-framework` package.
```

Reply:
Yeah, I agree. I also think "package" is redundant, given that we're in its README, so I'll remove that as well.

```suggestion
Check out the [Hello World tutorial](https://fluidframework.com/docs/start/tutorial/) using `fluid-framework`.
```